cypress/e2e/settings/data-import/check-return-to-default-in-suppress-from-discovery.cy.js

```diff
@@ -0,0 +1,205 @@
+import { including } from '@interactors/html';
+
+import { Permissions } from '../../../support/dictionary';
+import {
+  SettingsDataImport,
+  FieldMappingProfiles,
+  FieldMappingProfileView,
+} from '../../../support/fragments/settings/dataImport';
+import { SETTINGS_TABS } from '../../../support/fragments/settings/dataImport/settingsDataImport';
+import SettingsMenu from '../../../support/fragments/settingsMenu';
+import Users from '../../../support/fragments/users/users';
+import getRandomPostfix from '../../../support/utils/stringTools';
+
+describe('Data Import', () => {
+  describe('Settings', () => {
+    const testData = {
+      profileName: '',
+      user: {},
+    };
+
+    before('Create test user', () => {
+      cy.createTempUser([Permissions.settingsDataImportEnabled.gui]).then((userProperties) => {
+        testData.user = userProperties;
+      });
+    });
+
+    beforeEach('Login', () => {
+      testData.profileName = `autotest_mapping_profile_name_${getRandomPostfix()}`;
+
+      cy.login(testData.user.username, testData.user.password, {
+        path: SettingsMenu.dataImportSettingsPath,
+        waiter: SettingsDataImport.waitLoading,
+      });
+    });
+
+    afterEach('Delete mapping profile', () => {
+      cy.getAdminToken().then(() => {
+        FieldMappingProfiles.deleteMappingProfileByNameViaApi(testData.profileName);
```

Comment:
The function for deleting mapping profile already exists - FieldMappingProfileView.deleteViaApi(profileName);

Reply:
I propose to move API methods related with CRUD operation for import profiles in a separate PR